kt/godot-runtime/src/main/kotlin/godot/core/KtObject.kt

```diff
@@ -5,6 +5,21 @@ import godot.util.nullptr
 
 @Suppress("LeakingThis")
 abstract class KtObject {
+
+    /** Used to prevent the __new method to be executed when called from instantiateWith
+     * Instead we use the values set in that class  */
+    private class InitConfiguration {
+        var shouldOverride = false
+        var ptr: VoidPtr = 0
+        var id: Long = 0L
+
+        fun reset() {
+            shouldOverride = false
+            ptr = 0
+            id = 0
```

Comment:
so ids always start at 1?

Reply:
No, it starts at 0. I guess I should init it with -1